src/main/java/org/spongepowered/common/data/manipulator/mutable/entity/SpongeMovementSpeedData.java

```diff
@@ -84,8 +84,8 @@ protected void registerGettersAndSetters() {
     @Override
     public MutableBoundedValue<Double> walkSpeed() {
         return SpongeValueBuilder.boundedBuilder(Keys.WALKING_SPEED)
-            .defaultValue(0.1D)
-            .minimum(Double.MIN_VALUE)
+            .defaultValue(0.699999988079071D) // Why not 0.7D? Ask Mojang
```

Comment:
@ZephireNZ It's got this weird value because 0.7 can't be represented in binary without rounding, the same way 1/3 is 0.3333... in decimal.
This is what happens when decompiling the class, it is 0.7D in Mojang's code